src/debug/daccess/dacdbiimpl.cpp

```diff
@@ -1006,10 +1006,10 @@ void DacDbiInterfaceImpl::GetSequencePoints(MethodDesc *     pMethodDesc,
     // if there is a rejit IL map for this function, apply that in preference to load-time mapping
 #ifdef FEATURE_REJIT
     CodeVersionManager * pCodeVersionManager = pMethodDesc->GetCodeVersionManager();
-    NativeCodeVersion nativeCodeVersion = pCodeVersionManager->GetNativeCodeVersion(dac_cast<PTR_MethodDesc>(pMethodDesc), (PCODE)startAddr);
-    if (!nativeCodeVersion.IsNull())
+    ILCodeVersion ilVersion = pCodeVersionManager->GetNativeCodeVersion(dac_cast<PTR_MethodDesc>(pMethodDesc), (PCODE)startAddr).GetILCodeVersion();
```

Comment:
If nativeCodeVersion.IsNull() evaluated to true before, now the call to NativeCodeVersion::GetILCodeVersion() will AV. I'm not certain this code is called with input arguments that would lead to nativeCodeVersion.IsNull() == true, but its probably easier to protect against it than to do enough to code review to rule it out.

Reply:
Fixed